### PR TITLE
API 2302 Documentation Update for ArcGIS Removal

### PR DESCRIPTION
**Pull request by jnguy-va**
https://vajira.max.gov/browse/API-2302

Updated the PlantUML diagram and some notes in the README.

### CODE REVIEW

**Review by taylorrassmannlibertyits — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -46,10 +42,16 @@ OpenAPI documentation on the
   [CSV document](facilities/src/main/resources/websites.csv)
   of facility website URLS, maintained in this repository.
   This data is aggregated during facilities collection to augment the _live_ sources.
-* `Corporate Data Warehouse` undergoes a nightly ETL process to transfer
+* `Corporate Data Warehouse` undergoes a nightly ETL process to transfer various data resources to the _SQL52_ server, where it is consumed during facilities collection. Namely:
+    - Mental Health and Clinical Services from the _A01_ server.
+    - National Cemeteries and Benefit Centers from the _NCA VBA SQL40 GIS_ server.
+    - Health facilities and Veteran centers from the _A06_ server.
+    - Stop Codes from the _VSSC_ server.
+* `Corporate Data Warehouse` undergoes a nightly ETL process to transfer various data resources.
```
Comment (jblefkove-va) [inline README.md:50]: this bullet seems redundant now vs the one above

File: README.md [hunk 2]
```diff
@@ -46,10 +42,16 @@ OpenAPI documentation on the
   [CSV document](facilities/src/main/resources/websites.csv)
   of facility website URLS, maintained in this repository.
   This data is aggregated during facilities collection to augment the _live_ sources.
-* `Corporate Data Warehouse` undergoes a nightly ETL process to transfer
+* `Corporate Data Warehouse` undergoes a nightly ETL process to transfer various data resources to the _SQL52_ server, where it is consumed during facilities collection. Namely:
+    - Mental Health and Clinical Services from the _A01_ server.
+    - National Cemeteries and Benefit Centers from the _NCA VBA SQL40 GIS_ server.
+    - Health facilities and Veteran centers from the _A06_ server.
+    - Stop Codes from the _VSSC_ server.
```
Comment (jblefkove-va) [inline README.md:49]: Stop codes come from the clinical services (`App.VSSC_ClinicalServices`) table, so this bullet can be combined with the top one

File: src/plantuml/component-definitions.iuml [hunk 3]
```diff
@@ -30,10 +30,12 @@ database "Corporate Data Warehouse" as CDW <<dependency>>
...
 rectangle "Content Management System" as CMS <<partner>>
-rectangle "ArcGIS (VBA and NCA)" as ARCGIS <<dependency>>
 rectangle "Non-National\nCemeteries XML" as CXML <<document>>
 rectangle "Website CSV" as WCSV <<document>>
 rectangle "PSSG Drive Band" as BANDS <<document>>
-rectangle "ETL from A01 to SQL52" as ETL <<sql package>>
+rectangle "ETL from A01 to SQL52" as ETL1 <<sql package>>
+rectangle "ETL from SQL40 GIS Server to SQL52" as ETL2 <<sql package>>
+rectangle "ETL from A06 Raw Service to SQL52" as ETL3 <<sql package>>
+rectangle "ETL from VSSC to SQL52" as ETL4 <<sql package>>
```
Comment (jblefkove-va) [inline src/plantuml/component-definitions.iuml:39]: ETL4 can be combined with ETL1; they are both A01

**Comment by jnguy-va** [comment]
Think I got all the comments now

**Review by taylorrassmannlibertyits — APPROVED** [review]

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**